community/libgdata/APKBUILD

```diff
@@ -1,35 +1,34 @@
-# Contributor: 
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=libgdata
 pkgver=0.17.9
-pkgrel=1
+pkgrel=2
 pkgdesc="access library for gdata"
-url="http://live.gnome.org/libgdata"
+url="https://wiki.gnome.org/Projects/libgdata"
 arch="all"
-license="LGPL"
-makedepends="intltool uhttpmock-dev glib-dev 
-	libsoup-dev json-glib-dev liboauth-dev"
+license="LGPL-2.1-or-later"
+depends="gsettings-desktop-schemas"
+makedepends="intltool uhttpmock-dev glib-dev vala gobject-introspection-dev
+	libsoup-dev json-glib-dev liboauth-dev gnome-online-accounts-dev gcr-dev"
 checkdepends="gsettings-desktop-schemas"
+options="!check" #Needs to bind to ports for testing
```

Comment:
We have a lot of tests that bind to ports. I don't understand what goes wrong based on that comment.

Reply:
Hm, at least in my docker setup that caused it to fail, I assumed the same would happen in other environments. I'll dig out a log

Reply:
Yup, seems like it's just on docker, thanks for noticing.